src/Configuration/Config/src/ConfigurationRoot.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Extensions.Configuration
     /// <summary>
     /// The root node for a configuration.
     /// </summary>
-    public class ConfigurationRoot : IConfigurationRoot
+    public class ConfigurationRoot : IConfigurationRoot, IDisposable
```

Comment:
While you're here, also dispose the ChangeToken subscriptions.

Reply:
Good idea, I didn’t even see that before.

Reply:
That's prolly because he just added them recently :)